src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -535,6 +535,24 @@ def _get_components_pairs(
 
         return bat_inv_map, inv_bat_map
 
+    def _get_working_batteries(self, batteries: Set[int]) -> Set[int]:
+        """Get subset with working batteries.
+
+        If there are no working batteries in the given set,
+        then return input set to use them all in case of gap discharge.
+
+        Args:
+            batteries: requested batteries
+
+        Returns:
+            Subset with working batteries.
+            Input set if there are no working batteries in the given set.
```

Comment:
That looks like two values are returned.  I think out documentation tool treats that formatting as two return values as well.  Maybe something like this is better,  where you clearly say `or`
```suggestion
            Subset of working batteries, or entire input set if there are no working batteries in
                the given set.
```

Reply:
Done